data/dictionary/items getter/29a6f0f8-3472-4f31-8363-a9cdd575913d.json

```diff
@@ -0,0 +1,96 @@
+{
+  "description": "Retrieves multiple items' value from the dictionary with the proper index of the keys used for querying, in the order of the keys provided.\n\nExample A:\n1. { \"first\": 1, \"third\": 3, \"fifth\": 5 } @0 received via `dict`\n2. [ \"first\", \"third\" ]@0 received via `key`\n3. `items` sends  \n{\n  \"0\": 1,\n  \"2\": 3\n}@0\n\nExample B:\n1. { \"first\": 1, \"third\": 3, \"fifth\": 5 } @0 received via `dict`\n2. [\"first\",\"second\"]@0 received via `key`\n3. `items` sends  \n{\"0\": 1}@0\n\n",
+  "interface": {
+    "inputs": {
+      "257885b0-08c2-41c8-ba17-d6f9b80e93d1": {
+        "type": "{string:any}",
+        "name": "dict",
+        "description": "Receives the dictionary to get the items from.\n\nExample:\n{ \"first\": 1, \"third\": 3, \"fifth\": 5 }"
+      },
+      "72550b19-9434-43d7-a6af-468a19288808": {
+        "type": "string[]",
+        "name": "keys",
+        "description": "Receives the keys corresponding to the items to get.\n\nExample:\n[\"first\", \"third\"]"
+      }
+    },
+    "outputs": {
+      "6dd99f55-3d65-42ca-b301-c2c6367c334c": {
+        "type": "{string: any}",
+        "name": "items",
+        "description": "Sends the values with the index of the corresponding to the received keys.\n\nExample:\n{\n  \"0\": 1,\n  \"2\": 3\n}\n"
```

Comment:
The description is grammatically incorrect.
Maybe: "Sends the values with the index corresponding to the received keys."

Reply:
Done